src/components/Filters.js

```diff
@@ -7,7 +7,7 @@ const Filters = () => {
   const {
     filters: {
       text,
-      category,
+      category: currentCategory,
```

Comment:
Changing the variable name, so that it doesn't conflict/get shadowed by the `category` variable down on line 56